lightning/src/chain/chaininterface.rs

```diff
@@ -41,11 +41,11 @@ pub enum ConfirmationTarget {
 pub trait FeeEstimator {
 	/// Gets estimated satoshis of fee required per 1000 Weight-Units.
 	///
-	/// Must be no smaller than 253 (ie 1 satoshi-per-byte rounded up to ensure later round-downs
+	/// Must return a value no smaller than 253 (ie 1 satoshi-per-byte rounded up to ensure later round-downs
 	/// don't put us below 1 satoshi-per-byte).
 	///
-	/// This translates to:
-	///  * satoshis-per-byte * 250
+	/// This converts to:
```

Comment:
Converts seems weird to me? I've never heard "this converts to", only "this translates to" - maybe its a british-ism? Is "Conversions from other common units for this method are:" clearer?

Reply:
Hmm maybe, when I think of conversion I usually think unit conversion (liters into ounces etc.) and translation as some written or spoke (english to spanish etc)

Reply:
Right, I guess "This converts to" is just strange after a full sentence which "this" doesn't directly refer to, except in a general non-unit sense. Maybe just write it out into a sentence instead of "This translates to"

Reply:
Ok, let's leave as "translate" I don't feel strongly. We can keep an eye out for any further instances of confusion around this.

Reply:
I mean writing out a full sentence instead of three words is probably clearer for everyone, but I don't really care.

Reply:
Any clearer?

Reply:
How about something like "This method can be implemented with the following unit conversions"

Reply:
👍